cmd/datasource-syncer/Dockerfile

```diff
@@ -17,8 +17,8 @@ WORKDIR /app
 COPY . ./
 
 FROM buildbase as appbase
-RUN CGO_ENABLED=0 go build -mod=vendor -o datasource-syncer cmd/datasource-syncer/*.go
+RUN CGO_ENABLED=0 GOEXPERIMENT=boringcrypto go build -mod=vendor -o datasource-syncer cmd/datasource-syncer/*.go
 
 FROM gcr.io/distroless/static-debian11:latest
 COPY --from=appbase /app/datasource-syncer /bin/datasource-syncer
-ENTRYPOINT ["/bin/datasource-syncer"]
```

Comment:
nit (non-blocking): was this newline removed? Let's ensure we have file-ending newlines to ensure tools behave properly (and because it's enforced within Google).

Reply:
I don't think that's part of this change, but I added a newline.